### ISSUE TITLE
fix error in DecisionTrees.ipynb

### ISSUE THREAD

**Issue by vigsterkr** [shogun-toolbox/shogun#3756]
```
ERROR | Error while converting '/home/buildslave/nightly_default/build/doc/ipython-notebooks/multiclass/Tree/DecisionTrees.ipynb'
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/nbconvert/nbconvertapp.py", line 357, in export_single_notebook
    output, resources = self.exporter.from_filename(notebook_filename, resources=resources)
  File "/usr/local/lib/python2.7/dist-packages/nbconvert/exporters/exporter.py", line 165, in from_filename
    return self.from_file(f, resources=resources, **kw)
  File "/usr/local/lib/python2.7/dist-packages/nbconvert/exporters/exporter.py", line 183, in from_file
    return self.from_notebook_node(nbformat.read(file_stream, as_version=4), resources=resources, **kw)
  File "/usr/local/lib/python2.7/dist-packages/nbconvert/exporters/html.py", line 65, in from_notebook_node
    return super(HTMLExporter, self).from_notebook_node(nb, resources, **kw)
  File "/usr/local/lib/python2.7/dist-packages/nbconvert/exporters/templateexporter.py", line 200, in from_notebook_node
    nb_copy, resources = super(TemplateExporter, self).from_notebook_node(nb, resources, **kw)
  File "/usr/local/lib/python2.7/dist-packages/nbconvert/exporters/exporter.py", line 130, in from_notebook_node
    nb_copy, resources = self._preprocess(nb_copy, resources)
  File "/usr/local/lib/python2.7/dist-packages/nbconvert/exporters/exporter.py", line 302, in _preprocess
    nbc, resc = preprocessor(nbc, resc)
  File "/usr/local/lib/python2.7/dist-packages/nbconvert/preprocessors/base.py", line 47, in __call__
    return self.preprocess(nb,resources)
  File "/usr/local/lib/python2.7/dist-packages/nbconvert/preprocessors/execute.py", line 145, in preprocess
    nb, resources = super(ExecutePreprocessor, self).preprocess(nb, resources)
  File "/usr/local/lib/python2.7/dist-packages/nbconvert/preprocessors/base.py", line 70, in preprocess
    nb.cells[index], resources = self.preprocess_cell(cell, resources, index)
  File "/usr/local/lib/python2.7/dist-packages/nbconvert/preprocessors/execute.py", line 176, in preprocess_cell
    raise CellExecutionError(msg)
CellExecutionError: An error occurred while executing the following cell:
------------------
import matplotlib.pyplot as plt

cv_errors = [get_cv_error(i) for i in range(1,15)]
plt.plot(range(1,15),cv_errors,'bo',range(1,15),cv_errors,'k')
plt.xlabel('max_allowed_depth')
plt.ylabel('cross-validated error')
plt.ylim(0,1.2)
plt.show()
------------------

SystemError: [ERROR] In file /home/buildslave/nightly_default/build/src/shogun/evaluation/MeanSquaredError.cpp line 22: assertion predicted->get_label_type() == LT_REGRESSION fa$
led in virtual float64_t shogun::CMeanSquaredError::evaluate(shogun::CLabels*, shogun::CLabels*) file /home/buildslave/nightly_default/build/src/shogun/evaluation/MeanSquaredErr$
r.cpp line 22
```

**Comment by moizsajid** [comment]
I am working on this issue.

**Comment by my-zhang** [comment]
This issue is already resolved by #3771 Please close this ticket.

**Comment by karlnapf** [comment]
thx!
